src/eme.js

```diff
@@ -231,29 +276,7 @@ export const makeNewRequest = (player, requestOptions) => {
         if (expired) {
           // Close session and remove it from the session list to ensure that a new
           // session can be created.
-          videojs.log.debug('Session expired, closing the session.');
```

Comment:
We removed this debug statement and didn't add it back in `closeAndRemoveSession `.. should we add something similar?

Reply:
Good catch! Added it back in `closeAndRemoveSession`